src/box/lua/space.h

```diff
@@ -41,6 +41,10 @@ struct lua_State;
 void
 box_lua_space_init(struct lua_State *L);
 
+/** Free module resources. */
```

Comment:
Let's not add a comment - for consistency with `box_lua_space_init()`, which doesn't have one. Checkpatch doesn't require comments for init/free.

Reply:
Fixed.